nireports/reportlets/xca.py

```diff
@@ -359,8 +359,8 @@ def compcor_variance_plot(
         ax[m].set_yticks([])
         ax[m].set_yticklabels([])
         for tick in ax[m].xaxis.get_major_ticks():
```

Comment:
This feels cleaner to me:

```suggestion
        for label in ax[m].xaxis.get_majorticklabels():
```